specification/openai/resource-manager/Microsoft.OpenAI/preview/2021-09-01-preview/examples/CheckNameAvailability.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "api-version": "2021-09-01-preview",
```

Comment:
Do we have to include the "-preview" suffix if it's a preview service? Or it's optional?